<?xml version="1.0"?>
<launch>
    <!-- 启动底盘驱动 -->
    <include file="$(find autolabor_pro1_driver)/launch/driver.launch" />

    <!-- 启动Kinect2相机 -->
    <include file="$(find kinect2_bridge)/launch/kinect2_bridge.launch" />

    <!-- 加载参数 -->
    <param name="obstacle_distance_threshold" value="1.0" />
    <param name="safe_distance" value="0.5" />
    <param name="linear_speed" value="0.2" />
    <param name="angular_speed" value="0.2" />

    <!-- 启动变道控制节点 -->
    <node name="lane_change_controller" pkg="lane_change_control" type="lane_change_node" output="screen">
    </node>
</launch>
